src/test/regress/expected/multi_extension.out

```diff
@@ -77,6 +77,8 @@ END
 $func$ LANGUAGE plpgsql;
 CREATE SCHEMA test;
 :create_function_test_maintenance_worker
+WARNING:  Citus can't distribute functions having dependency on unsupported relation with relkind v
```

Comment:
cool that we catch this :) The function has dependency to pg_stat_activity via RETURN clause.

We might have set citus.enable_metadata_sync to off;  but, this is fine for now, we have to change this when we support distributed VIEWs